### PR TITLE
Added

### PR DESCRIPTION
**Pull request by shanti-s**
Please check the exercise

### CODE REVIEW

File: exercise1.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+console.log(firstName, lastName);
+
+const fName = "Dave";
+const lName = "Williams";
+const city = "Berlin";
+const job = "Teacher";
+const age = "31";
+console.log(fName, lName, city, job, age);
+
+console.log("John Smith " + "is " + "a " + "43 " + "year " +
```
Comment (ghassanaldarwish) [inline exercise1.js]: we can use the variable above

console.log(fName+ lName + "is " ...)

**Review by ghassanaldarwish — COMMENTED** [review]
we can use the variable above

console.log(fName+ lName + "is " ...)


console.log("John Smith " + "is " + "a " + "43 " + "year " ..)

File: exercise1.js (hunk 1 above)
Comment (shanti-s) [reply exercise1.js]: Got it. Thank You

**Review by ghassanaldarwish — APPROVED** [review]
good job

**Pull request closed without merging**